src/ORM/Query.php

```diff
@@ -874,7 +874,6 @@ public function applyOptions(array $options)
     public function cleanCopy()
     {
         $clone = clone $this;
-        $clone->triggerBeforeFind();
```

Comment:
We'll need to document this as this change could impact other usage of `cleanCopy()` in userland code.